src/dbcp/transform/gridstatus.py

```diff
@@ -671,14 +756,19 @@ def _transform_nyiso(iso_df: pd.DataFrame) -> pd.DataFrame:
     ).all(), "Some projects are marked marked actionable and nearly certain."
 
     iso_df["interconnection_status_raw"] = iso_df["S"].map(status_mapping)
+
+    # iso_df = iso_df.rename(
+    #     columns={"Developer Name": "developer"}, errors="raise"
+    # )
```

Comment:
[GS incorrectly handled this column](https://github.com/kmax12/gridstatus/pull/370). Working on updating the archives.

Reply:
We need it to flag offshore wind projects.